web/src/settings_invites.ts

```diff
@@ -164,9 +165,10 @@ function do_revoke_invite({
             }),
             $("#home-error"),
         );
+        dialog_widget.hide_dialog_spinner();
```

Comment:
This should be in the commit which changes the error to be shown in the modal. Also, I guess would be good to have that as a first commit since that contains the changes expected in the issue.

Reply:
The first commit should only contain changes for showing the errors in dialog instead of `home-error` element and no other changes.